.travis.yml

```diff
@@ -1,29 +1,30 @@
+<<<<<<< HEAD
+=======
+# Work-around for https://github.com/travis-ci/travis-ci/issues/8836#issuecomment-356362524.
+# (Restore `sudo: false` once that is resolved.)
+sudo: required
+>>>>>>> Replaced nyc by karma
```

Comment:
Merge conflict?

Reply:
Yes my bad, thanks for reporting!

Also, because the bug affected `.travis.yml`, Travis build didn't even start (and therefore didn't report any error back to the PR). Because CodeClimate and Snyk ran independently and succeeded, PR status was green.

Reply:
...and fixed ;)

Reply:
... and added build status badge to PR description